jenkins/bootstrap.py

```diff
@@ -770,6 +770,7 @@ def setup_root(call, root, repos, ssh, git_cache, clean):
         os.makedirs(root)
     root_dir = os.path.realpath(root)
     logging.info('Root: %s', root_dir)
+    os.environ['GIT_TRACE'] = '1'
```

Comment:
And yes it will pollute os.environ for the duration of the bootstrap.py call (but not when it returns back to you)